src/components/transactionsV2/transactionsOverviewHeader/transactionsOverviewHeader.js

```diff
@@ -115,7 +115,9 @@ class transactionsHeader extends React.Component {
             )}
             <DropdownV2
               showDropdown={this.state.shownDropdown === 'followDropdown'}
-              className={'follow-dropdown'}></DropdownV2>
+              className={'follow-dropdown'}>
+                <div></div>
+              </DropdownV2>
```

Comment:
what exactly is doing this line? I dont know if what a mistake or not

Reply:
Just to fix a warning on the DropdownV2 component,